serf/serf.go

```diff
@@ -785,11 +785,12 @@ func (s *Serf) Members() []Member {
 // immediately, instead of waiting for the reaper to eventually reclaim it.
 // This also has the effect that Serf will no longer attempt to reconnect
 // to this node.
-func (s *Serf) RemoveFailedNode(node string) error {
+func (s *Serf) RemoveFailedNode(node string, prune bool) error {
```

Comment:
Same comment here about breaking the arity of this function. I know this will be a breaking change in nomad.